app/assets/stylesheets/components/_navbarRFB.scss

```diff
@@ -10,26 +10,26 @@
 
 }
 
-.container {
-    padding-left: 6px;
-    padding-right: 6px;
-    margin-right: auto;
-    margin-left: auto;
-    height: 78px;
-}
-
-@media (min-width: 1200px) {
-    .container {
-        width: 100%;
-    }
-}
+// .container {
```

Comment:
I had to comment this class. Since it we are the .container class from bootstrap in other front-end files, if we override the original .container, it affects others areas of the system . So, i commented this class!!